templates/tagging_common.mako

```diff
@@ -77,7 +77,7 @@ from markupsafe import escape
 
         item_tag_names = []
         for ta in item_tags:
-            item_tag_names.append(escape(ta.tag.name))
+            item_tag_names.append(escape(f"#{ta.value}" if ta.value else ta.tag.name))
```

Comment:
Sorry, I just looked at this again ... I don't think this is right. To match the API this should be `item.make_tag_string_list()`. If the rendering is off that should then be handled on the client side, probably ?

Reply:
Argh.  Yeah, there's more complexity here, and more issues with tag values.

i.e. -- I can have a tag `my.hierarchical.tag:myvalue`  and now this isn't right.

Reply:
Maybe we just try to get it right in dev with the API and see what we'd need to backport ?

Reply:
Yeah.  The API shouldn't be doing any munging of `name`, `#`, or anything like that if it is currently.  The client should do all that.